plugin/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+	"root": true,
+	"extends": "wpcalypso",
+	"parser": "babel-eslint",
+	"env": {
+		"browser": true,
+		"node": true
+	},
+	"parserOptions": {
+		"ecmaVersion": 3
```

Comment:
From the docs:

> You only need to use babel-eslint if you are using types (Flow) or experimental features not supported in ESLint itself yet

We may need it, if we decide to add object-spread, but we could drop it for now.